packages/sdks/src/blocks/columns/columns.lite.tsx

```diff
@@ -74,57 +81,124 @@ export default function Columns(props: ColumnProps) {
         '--column-margin-left-tablet': state.maybeApplyForTablet(marginLeft),
       };
     },
+
+    getWidthForBreakpointSize(size: SizeName) {
+      const breakpointSizes = getSizesForBreakpoints(
+        props.customBreakpoints || {}
+      );
+
+      return breakpointSizes[size].max;
+    },
+
+    get columnsStyles(): string {
+      return `
+        @media (max-width: ${state.getWidthForBreakpointSize('medium')}px) {
+          .${props.builderBlock.id}-breakpoints {
+            flex-direction: var(--flex-dir-tablet);
+            align-items: stretch,
+          }
+
+          .${props.builderBlock.id}-breakpoints > .builder-column {
+            width: var(--column-width-tablet) !important;
+            margin-left: var(--column-margin-left-tablet) !important;
+          }
+        }
+
+        @media (max-width: ${state.getWidthForBreakpointSize('small')}px) {
+          .${props.builderBlock.id}-breakpoints {
+            flex-direction: var(--flex-dir);
+            align-items: stretch;
+          }
+
+          .${props.builderBlock.id}-breakpoints > .builder-column {
+            width: var(--column-width) !important;
+            margin-left: var(--column-margin-left) !important;
+          }
+        },
+      `;
+    },
   });
 
   return (
-    <div
-      class="builder-columns"
-      css={{
-        display: 'flex',
-        alignItems: 'stretch',
-        lineHeight: 'normal',
-        '@media (max-width: 991px)': {
-          flexDirection: 'var(--flex-dir-tablet)',
-        },
-        '@media (max-width: 639px)': {
-          flexDirection: 'var(--flex-dir)',
-        },
-      }}
-      style={state.columnsCssVars as CSS}
+    <Show
+      when={TARGET !== 'reactNative'}
+      else={
```

Comment:
@samijaber I have updated the PR to not use the div above and not use 2 SHOW components, instead a single one with 'else' like we discussed yesterday